pymatgen/io/qchem/inputs.py

```diff
@@ -97,7 +96,25 @@ def __init__(
             geom_opt (dict):
                     A dictionary of input parameters for the geom_opt section of the QChem input file.
                     This section is required when using the new libopt3 geometry optimizer.
-
+            svp (dict):
+                A dictionary of all the input parameters for the svp section of QChem input file. This section
+                is required to use the isodensity SS(V)PE and CMIRS solvation models. Note that for the CMIRS model,
+                the RHOISO value must be coordinated with the solvent parameters specified in the $pcm_nonels section.
+                svp parameters should be passed as a dict, e.g.
+                svp={"RHOISO": 0.001, "DIELST": 78.39, "NPTLEB": 110}
+                Note that the "DIELST" setting has no effect on non-isodensity SS(V)PE calculations (i.e., when
+                solvent_method = PCM).
+            pcm_nonels (dict):
```

Comment:
This name feels misleading and possibly just wrong. These aren't PCM parameters. They're nonelectrostatic implicit solvent parameters for CMIRS.

Reply:
That's fair; however `$pcm_nonels` is the name Q-Chem chose for the relevant input file section. It seems like throughout our infrastructure we use the exact section names as variables (e.g. `rem`, `solvent`, etc.) to avoid ambiguity, so I thought it best to do so here as well.